src/windowsTerminal.ts

```diff
@@ -119,9 +119,14 @@ export class WindowsTerminal extends Terminal {
     this._writable = true;
 
     this._forwardEvents();
+  }
+
+  protected _write(data: string): void {
+    this._defer(this._doWrite, data);
```

Comment:
Do we need to `this._doWrite.bind(this)` here? Should _doWrite be a property that is set in the ctor because of that?

Reply:
Currently not needed since both live in the same class scope.